app/models/like.rb

```diff
@@ -0,0 +1,10 @@
+class Like < ApplicationRecord
+  belongs_to :post, class_name: 'Post'
+  belongs_to :author
```

Comment:
[REQUIRED] Kindly update this to `belongs_to :author, class_name: 'User'`.  So that rails can know that likes is referencing the user table